dockerfiles/onnxruntime-tensorrt.dockerfile

```diff
@@ -7,21 +7,37 @@
 # Dockerfile to run ONNXRuntime with TensorRT integration
 
 # nVidia TensorRT Base Image
-FROM nvcr.io/nvidia/tensorrt:20.07.1-py3
+FROM nvcr.io/nvidia/tensorrt:20.07.1-py3 AS runtime
 
 ARG ONNXRUNTIME_REPO=https://github.com/Microsoft/onnxruntime
 ARG ONNXRUNTIME_BRANCH=v1.5.3
 
-RUN apt-get update &&\
-    apt-get install -y sudo git bash unattended-upgrades
-
 WORKDIR /code
 ENV PATH /usr/local/nvidia/bin:/usr/local/cuda/bin:/code/cmake-3.14.3-Linux-x86_64/bin:/opt/miniconda/bin:${PATH}
 ENV LD_LIBRARY_PATH /opt/miniconda/lib:$LD_LIBRARY_PATH
 
+RUN DEBIAN_FRONTEND=noninteractive \
+    apt-get update && apt-get install -y --no-install-recommends \
+            wget \
+            zip \
+            ca-certificates \
+            build-essential \
+            curl \
+            libcurl4-openssl-dev \
+            libssl-dev \
+            python3-dev
+
+RUN wget --quiet https://repo.anaconda.com/miniconda/Miniconda3-4.5.4-Linux-x86_64.sh -O ~/miniconda.sh --no-check-certificate && /bin/bash ~/miniconda.sh -b -p /opt/miniconda && \
```

Comment:
I think it would be best to use the same installation process as in the [main docker file](https://github.com/nodefluxio/vortex/blob/652d04705acdf92768ad36f2a347de5f1e75bc3f/Dockerfile#L19) for consistency. and also it would be better to install pytorch in here using conda including the corresponding cudatoolkit version to avoid errors as you have suggested before.

Reply:
Ok

Reply:
last time I tried using latest miniconda the installation for development will breaks, thats why it uses this version

Reply:
but ok let me try again

Reply:
same actually with the main docker, but I use `--ignore-installed` to deal with it, just as a workaround really.